melange.yaml

```diff
@@ -122,6 +122,43 @@ subpackages:
         - runs: |
             usrmerge-tool --help
 
+  - name: sht
+    options:
+      no-provides: true
+    dependencies:
+      runtime:
+        - go-test2json
```

Comment:
this seems like it shouldn't be necessary. surely go-test2json is importable in go, no?

Reply:
that is surprising to me that its not importable. yea, i kind of feel like ship it inside. but it doesn't matter too much.